clang/lib/Sema/SemaOverload.cpp

```diff
@@ -2268,17 +2268,15 @@ static bool IsStandardConversion(Sema &S, Expr* From, QualType ToType,
     // handling here.
     if (ToType->isArrayParameterType()) {
       FromType = S.Context.getArrayParameterType(FromType);
-      SCS.First = ICK_HLSL_Array_RValue;
     } else if (FromType->isArrayParameterType()) {
       const ArrayParameterType *APT = cast<ArrayParameterType>(FromType);
```

Comment:
nit:
```suggestion
    } else if (const ArrayParameterType *APT = dyn_cast<ArrayParameterType>(FromType)) {
```
I think this is equivalent? The function seems to just be a wrapper to the `isa<..>` equivalent

Reply:
This looks like its just an NFC change to code I didn't touch, so I think I'll leave it as is.